functions.php

```diff
@@ -14,23 +20,27 @@ function extractFromDB(PDO $db): array
     return $records;
 }
 
+//Creates a string of concatenated information from the database to display
+//@param $array array the data extracted from extractFromDB($db)
+//@return string to display
```

Comment:
I like these docblocks!! They're really clear, and I like the technical language you're using. 
For future reference they're usually in this specific format:
```
/**
  * Creates a string of concatenated information from the database to display
  * @param $array array the data extracted from extractFromDB($db)
  * @return string to display
**/
```
Nice work!